Lib/xml/etree/ElementTree.py

```diff
@@ -1200,6 +1200,13 @@ def parse(source, parser=None):
     Return an ElementTree instance.
 
     """
+    if (getattr(source, 'encoding', None) == 'ISO-8859-1' and
+        source.mode == 'r'):
+        import warnings
+        warnings.warn(
+            "For file objects containing XML data"
+            "with non-ASCII and non-UTF-8 encoding (e.g. ISO 8859-1), "
```

Comment:
We try to avoid latin abbreviations. Please spell it out in plain english.

Reply:
It seems that e.g. is more concise than for example (although, I agree with your suggestion).